commands/dns_cli.py

```diff
@@ -324,9 +324,15 @@ def check_r53(domain_session, project_session, domain, base_domain):
 def get_load_balancer_domain_and_configuration(
     project_session: Session, app: str, svc: str, env: str
 ) -> Tuple[str, dict]:
-    def separate_hyphenated_application_environment_and_service(hyphenated_string):
-        # The application name may be hyphenated, so we start splitting at the second from last occurrence of a hyphen
-        return hyphenated_string.rsplit("-", 2)
+    def separate_hyphenated_application_environment_and_service(
+        hyphenated_string, number_of_items_of_interest, number_of_trailing_items
+    ):
+        # The application name may be hyphenated, so we start splitting
+        # at the hyphen after the first item of interest and return the
+        # items of interest only...
+        rsplit = hyphenated_string.rsplit("-", number_of_trailing_items + number_of_items_of_interest - 1)
```

Comment:
Not a blocker, but these variable names are a bit of a mouthful

Reply:
But you don't have to think about what they mean ;-)

I'm open to suggestions for conveying the same meaning in less words.